frontend/src/app/homepage-offer/offers.service.ts

```diff
@@ -10,35 +10,39 @@ import { environment } from '../../environments/environment';
 
 @Injectable()
 export class OffersService {
-  private url = `${environment.apiRoot}/offers/`;
-  constructor (private http: Http) { }
+  private _url = `${environment.apiRoot}/offers/`;
+  constructor(private http: Http) { }
 
   getOffers() {
-    return this.http.get(this.url, { withCredentials: true } )
+    return this.http.get(this._url, { withCredentials: true })
       .map((res: Response) => res.json())
       .map(offers => {
         for (const offer of offers) {
-          this.loadDefaultImage(offer);
+          this._loadDefaultImage(offer);
         }
         return offers;
       });
   }
 
   getOffer(id: number): Observable<Offer> {
-    return this.http.get(`${this.url}${id}/`, { withCredentials: true })
+    return this.http.get(`${this._url}${id}/`, { withCredentials: true })
       .map((res: Response) => res.json())
-      .map(offer => this.loadDefaultImage(offer));
+      .map(offer => this._loadDefaultImage(offer));
   }
 
-  loadDefaultImage(offer: Offer): Offer {
-    if (offer.image === null) {
-        offer.image = 'assets/img/banner/volontulo_baner.png';
+  _loadDefaultImage(offer: Offer): Offer {
```

Comment:
if you name it private-like, then please make it `private`. However, since this `_` or not to `_` is purely opinion-based and angular styleguide discourages that, I'd personally keep the name as it was.